<launch>
      <!-- Initialize the Onboard Setpoint Transformer -->
      <node pkg="bridge_px4" type="setpoint_transform.py" name ="setpoint_transform" output="screen">
            <param name="drone_id" value="ASGARD"/>
            <param name="x_off" value="0.00"/>
            <param name="y_off" value="0.00"/>
            <param name="z_off" value="0.50"/>
      </node>
</launch>